public/js/services/game.js

```diff
@@ -87,6 +89,14 @@ angular.module('mean.system')
 
     var newState = (data.state !== game.state);
 
+    
+    game.chatUsername = data.players[game.playerIndex].username;
+    game.gameChat.setChatAvatar(data.players[game.playerIndex].avatar);
+    game.gameChat.setChatUsername(game.chatUsername);
+    game.gameChat.setChatGroup(data.gameID);
+    game.gameChat.listenForMessages();
+
+
     //Handle updating game.time
     if (data.round !== game.round && data.state !== 'awaiting players' &&
```

Comment:
Declare a variable to hold the states you are looking out for like this. 
```javascript
const undesiredGameStates = ['awaiting players', 'game ended', 'game dissolved'];

if(data.round !== game.round && !undesiredGameStates.includes(data.round))
```

The advantage of doing this is that it makes it easier to manage/maintain the conditions